subgraphs/compound-forks/protocols/bastion-protocol/src/mapping.ts

```diff
@@ -157,6 +188,10 @@ export function handleMarketListed(event: MarketListed): void {
     ),
     event
   );
+
+  const market = Market.load(cTokenAddr.toHexString())!;
```

Comment:
why do we need this 2x?

Reply:
If we only need it once then we would probably want to put the naming if block in the same if block to reduce the scope of those vars

Reply:
We need both, because the earlier chunk is for native token, and it returns at [line 159](https://github.com/tnkrxyz/subgraphs/blob/abbf59ee3cc00a74c4a74f467b8e62aa9fa2d7db/subgraphs/compound-forks/protocols/bastion-protocol/src/mapping.ts#L159)